Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -1751,6 +1751,122 @@ script:
       command with argument all=true. Do note that scanning all the endpoints may cause performance issues and latency.
     execution: true
     name: xdr-endpoint-scan
+    outputs:
+    - contextPath: PaloAltoNetworksXDR.endpointScan.actionId
+      description: The action ID of the scan request.
+      type: Number
+    - contextPath: PaloAltoNetworksXDR.endpointScan.aborted
+      description: Was the scan aborted.
+      type: Boolean
+  - arguments:
+    - default: false
+      description: List of endpoint IDs.
+      isArray: true
+      name: endpoint_id_list
+      required: false
+      secret: false
+    - default: false
+      description: Name of the distribution list.
+      isArray: true
+      name: dist_name
+      required: false
+      secret: false
+    - default: false
+      description: Integer in timestamp epoch milliseconds.
```

Comment:
I'd consider rephrasing

Reply:
@amshamah419 Did you look at the YML as well? Maybe I'm missing something here?

Reply:
I usually don't unless specified. Regarding this, I would use `Epoch timestamp in milliseconds represented as an integer.`

Reply:
done